php/sync/class-download-sync.php

```diff
@@ -293,10 +293,17 @@ public function import_asset( $attachment_id, $file_path, $transformations = nul
 		if ( 'image' === $attachment['type'] ) {
 			// Get the cloudinary_id from public_id not Media::cloudinary_id().
 			$cloudinary_id = $this->plugin->components['media']->get_cloudinary_id( $attachment_id );
+
+			// don't apply the default transformations here.
+			add_filter( 'cloudinary_apply_default_transformations', '__return_false' );
+
 			// Make sure all sizes have the transformations on for previewing.
 			foreach ( $attachment['sizes'] as $name => &$size ) {
-				$size['url'] = $this->plugin->components['media']->cloudinary_url( $attachment_id, $name, $transformations, $cloudinary_id );
+				$size['url'] = $this->plugin->components['media']->cloudinary_url( $attachment_id, $name, $transformations, $cloudinary_id, false );
```

Comment:
@dugajean 

Same here. It seems that we not need the last argument here. Can you please confirm?